tests/UnifiedSpecTests/Operation.php

```diff
@@ -491,7 +491,11 @@ function (IndexInfo $info) {
                 assertArrayHasKey('to', $args);
                 assertIsString($args['to']);
 
-                return $collection->rename($args['to']);
+                return $collection->rename(
+                    $args['to'],
+                    null,
+                    array_diff_key($args, ['to' => 1])
+                );
```

Comment:
These changes were made to make able RenameCollection helper receive "dropTarget" option, which is needed for new test.